php/commands/scaffold.php

```diff
@@ -446,6 +455,8 @@ function plugin( $args, $assoc_args ) {
 
 		if ( isset( $assoc_args['activate'] ) ) {
 			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug ) );
+		} else if ( isset( $assoc_args['activate-network'] ) ) {
+			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug), array( 'network' => true ) );
```

Comment:
Changes to plugin activation should be in a separate pull request from those to theme activation.